### PR TITLE
Recent connections menu is missing in lxqt-panel

### PR DESCRIPTION
**Pull request by niknah**
In lxqt-panel.  GetLayout is called twice.  Once with parent_id = 0, then parent_id = 7(recent connections).  But it doesn't rebuild the recent connection submenu only the menu item.
I am using lxqt-panel v1.1.0

### CODE REVIEW

File: blueman/main/indicators/StatusNotifierItem.py [hunk 1]
```diff
@@ -36,7 +36,11 @@ def _get_layout(self, parent_id: int, _recursion_depth: int, _property_names: Li
         if parent_id == 0:
             return self._revision, (0, {}, self._render_menu(self._items, 1, self._render_submenu))
         else:
-            return self._revision, (parent_id, self._render_item(self._items[parent_id - 1]), [])
+            item = self._items[parent_id - 1]
+            if "submenu" in item:
+                return self._revision, (parent_id, {}, self._render_submenu(item, parent_id ))
```
Comment (cschramm) [inline blueman/main/indicators/StatusNotifierItem.py:41]: Are you sure `{}` is correct here? Shouldn't it just be
```
return self._revision, (parent_id, self._render_item(self._items[parent_id - 1]), self._render_submenu(item, parent_id))
```
for all `parent_id != 0`?

Comment (niknah) [reply blueman/main/indicators/StatusNotifierItem.py]: Yes, you're right.  The audio profile menu had disappeared with that change.

Comment (cschramm) [reply blueman/main/indicators/StatusNotifierItem.py]: There's no need for the `else` branch then as `_render_submenu` handles items without submenus just fine.

Also there's a whitespace error that makes mypy unhappy.

Comment (niknah) [reply blueman/main/indicators/StatusNotifierItem.py]: Thanks, I have changed this and added a check for the _recursion_depth parameter

**Review by cschramm — APPROVED** [review]

**Pull request merged**